### PR TITLE
GoogleAutoComplete to support new RN >=60

### PR DESCRIPTION
**Pull request by pavan88**
To support RN >=60 and resolve the issue "TypeError: undefined is not an object (evaluating 'navigator.geolocation.GetCurrentPosition'

### CODE REVIEW

**Comment by adtm** [comment]
I see 4 PR's for this thing, could someone finally merge?

**Comment by xlogix** [comment]
@FaridSafi

**Comment by ntcong91** [comment]
+1 - @FaridSafi

File: GooglePlacesAutocomplete.js [hunk 1]
```diff
@@ -188,7 +189,7 @@ export default class GooglePlacesAutocomplete extends Component {
       }
     }
 
-    navigator.geolocation.getCurrentPosition(
+    Geolocation.getCurrentPosition(
       (position) => {
         if (this.props.nearbyPlacesAPI === 'None') {
           let currentLocation = {
```
Comment (sapter) [inline GooglePlacesAutocomplete.js:195]: In the second argument to `getCurrentPosition`, the callback `(error) => {this..., alert...}` change `alert` to `Alert.alert`, as `alert` itself is no longer a global variable in RN and needs to be imported via RN (i.e. `import {Alert} from 'react-native`)

File: GooglePlacesAutocomplete.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ import {
 } from 'react-native';
```
Comment (sapter) [inline GooglePlacesAutocomplete.js:17]: Add `Alert` to RN imports and see comment below

**Comment by bell-steven** [comment]
Thanks for the PR!

Closed in favor of #553.

**Pull request closed without merging**